src/rt64lib/public/rt64.h

```diff
@@ -271,6 +329,7 @@ typedef void (*SetViewDescriptionPtr)(RT64_VIEW *viewPtr, RT64_VIEW_DESC viewDes
 typedef void (*SetViewSkyPlanePtr)(RT64_VIEW *viewPtr, RT64_TEXTURE *texturePtr);
 typedef RT64_INSTANCE* (*GetViewRaytracedInstanceAtPtr)(RT64_VIEW *viewPtr, int x, int y);
 typedef bool (*GetViewFeatureSupportPtr)(RT64_VIEW *viewPtr, int feature);
+typedef void (*SetDeltaTimePtr)(RT64_VIEW* viewPtr, float deltaMs);
```

Comment:
This is more along the lines of what I expected, I'd just make a few changes:

1) Make this a parameter for DrawDevice instead. Since they both go hand in hand it won't very useful to have the parameters split.
2) Make it use seconds instead of ms (calling it `deltaTime` wil be enough for that). This is just for consistency.
3) Naturally, the SetDeltaTime function wouldn't be needed anymore once the functionality is merged into `DrawDevice`.